contactsummary/js/src/contact-summary/controllers/key-dates.controller.js

```diff
@@ -87,7 +87,7 @@ define([
      * @return {boolean}
      */
     function isDateInFuture (date) {
-      return moment().diff(date) < 0;
+      return moment().diff(date, 'days') <= 0;
```

Comment:
This is not future anymore, is this **today or future** now? Why has this change been done? What the problem was?

Reply:
Few points:
1. if the job role end date is less than today -> considered as past (not active)
2. **if the job role end date is  today -> considered as present (active)**
3. if the job role end date is greater than today -> considered as present (active)
4. if the job role end date is empty -> considered as present (active)

Previously, the above logic was not meeting the point no 1. So this change has been done.

Reply:
Then the function name is not correct isDateInFuture. Today !== Future!